### PR TITLE
Update tools help and man pages

### LINKED ISSUE #3731: Update `man` pages for Rizin tools
For every tool/binary Rizin provides `man` page should be updated to be in sync with the actual options and features.
Also, for some tools there are no man pages at all, e.g. `rz-test`.
- [x] rizin
- [x] rz-diff
- [ ] rz-asm
- [ ] rz-ax
- [ ] rz-bin
- [ ] rz-find
- [ ] rz-gg
- [ ] rz-hash
- [ ] rz-run
- [x] rz-sign
- [ ] rz-test

### PR DESCRIPTION
**Pull request by byteninjaa0**
# SQUASH ME

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Update man pages to reflect the current state of Rizin tools.

**Test plan**

- CI is green
- See if changes make sense

**Closing issues**

Closes #3731

### CODE REVIEW

File: binrz/man/rz-bin.1 [hunk 1]
```diff
@@ -1,95 +1,92 @@
...
-.Op Fl O Ar binop
-.Op Fl o Ar str
+.Op Fl C Ar F:C:D
+.Op Fl f Ar str
 .Op Fl m Ar addr
-.Op Fl @ Ar addr
 .Op Fl n Ar str
-.Op Fl X Ar fmt file ...
+.Op Fl N Ar m:M
+.Op Fl P[-P] pdb
```
Comment (wargio) [inline binrz/man/rz-bin.1]: this looks weird.

File: binrz/man/rz-gg.1 [hunk 2]
```diff
@@ -16,87 +17,88 @@
 .Op Fl B Ar hexpairs
 .Op Fl c Ar k=v
 .Op Fl C Ar file
-.Op Fl n Ar num32
-.Op Fl N Ar num64
+.Op Fl n Ar dword
+.Op Fl N Ar dword
```
Comment (wargio) [inline binrz/man/rz-gg.1:21]: change these to `value`

File: binrz/man/rz-run.1 [hunk 3]
```diff
@@ -1,124 +1,85 @@
-.Dd Feb 3, 2017
+.Dd Jan 24, 2024
 .Dt RZ_RUN 1
 .Sh NAME
 .Nm rz-run
 .Nd rizin utility to run programs in exotic environments
 .Sh SYNOPSIS
 .Nm rz-run
+.Op -v | -t | script.rr2
```
Comment (wargio) [inline binrz/man/rz-run.1]:
```suggestion
.Op -v | -t | script.rz
```

Comment (wargio) [reply binrz/man/rz-run.1]: if we have still `rr2` in the help, please add a commit changing this.

**Review by wargio — COMMENTED** [review]
wow nice changes!!

File: binrz/man/rz-gg.1 (hunk 2 above)
Comment (wargio) [reply binrz/man/rz-gg.1:21]: please submit a patch for the help command of this tool and change dword to `value`

**Review by wargio — APPROVED** [review]

File: binrz/man/rz-asm.1 [hunk 4]
```diff
@@ -1,108 +1,78 @@
...
 .Sh DESCRIPTION
 This tool uses r_asm to assemble and disassemble files or hexpair strings. It supports a large list of architectures which can be listed using the \-L flag.
 .Pp
 .Bl -tag -width Fl
 .It Fl a Ar arch
-Set architecture plugin
+Set architecture to assemble/disassemble (see -L)
 .It Fl A
-Show analysis information of given hexpair string
+Show Analysis information from given hexpairs
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: just "analysis", no need to capitalize

File: binrz/man/rz-asm.1 [hunk 5]
```diff
@@ -1,108 +1,78 @@
...
 .It Fl b Ar bits
-Set architecture bits
+Set cpu register size (8, 16, 32, 64) (RZ_ASM_BITS)
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: "CPU"

File: binrz/man/rz-asm.1 [hunk 6]
```diff
@@ -1,108 +1,78 @@
...
 .It Fl B
-Binary input/output (\-l is mandatory for binary input)
+Binary input/output (-l is mandatory for binary input)
 .It Fl c Ar cpu
-Select specific CPU (depends on \-a arch)
+Select specific CPU (depends on arch)
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: "depends on the arch"

File: binrz/man/rz-asm.1 [hunk 7]
```diff
@@ -1,108 +1,78 @@
...
 Specify input and/or output filters (att2intel, x86.pseudo, ...)
-.It Fl h
-Show usage help message.
-.It Fl hh
-Show long help message including supported assembler directives
-.It Fl l Ar int
+.It Fl h, hh
+Show this help, -hh for long.
+.It Fl i Ar len
+ignore/skip N bytes of the input buffer
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: "Ignore" to be consistent with other options

File: binrz/man/rz-asm.1 [hunk 8]
```diff
@@ -1,108 +1,78 @@
...
+.It Fl j
+output in json format
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: Same, also "JSON"

File: binrz/man/rz-asm.1 [hunk 9]
```diff
@@ -1,108 +1,78 @@
...
-Ignore/skip N bytes from the beginning of the input buffer
 .It Fl L
-List supported asm plugins
-.It Fl o Ar offset
-Offset of the opcode to assemble (default is 0)
-.It Fl @ Ar offset
-Alias for -o
-.It Fl O Ar ofile
-output to file, for example 'rz-asm \-BF a a.asm'
+List Asm plugins: (a=asm, d=disasm, A=analyze, e=ESIL)
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: "asm"

File: binrz/man/rz-asm.1 [hunk 10]
```diff
@@ -1,108 +1,78 @@
...
+.It Fl o, @ Ar addr
+Set start address for code (default 0)
+.It Fl O Ar file
+Output file name (rz-asm -Bf a.asm -O a)
+.It Fl p
+Run SPP over input for assembly
+.It Fl q
+quiet mode
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: "Quiet"

File: binrz/man/rz-asm.1 [hunk 11]
```diff
@@ -1,108 +1,78 @@
...
 .It Fl r
-Show output in rizin script
+output in rizin commands
```
Comment (XVilka) [inline binrz/man/rz-asm.1]: "Output"

File: binrz/man/rz-ax.1 [hunk 12]
```diff
@@ -1,100 +1,100 @@
...
 .Op [expr] ...
 .Sh DESCRIPTION
 This command is part of the rizin project.
 .Pp
-This command allows you to convert values between positive and negative integer, float, octal, binary and hexadecimal values.
+This command allows you to convert values between positive and negative integers, floats, octal, binary, ternary, and hexadecimal values.
 .Sh OPTIONS
 .Bl -tag -width Fl
+.It Fl l
+append newline to output (for -E/-D/-r/..)
```
Comment (XVilka) [inline binrz/man/rz-ax.1]: Same

File: binrz/man/rz-ax.1 [hunk 13]
```diff
@@ -1,100 +1,100 @@
...
+.It Fl f
+Floating point arithmetic (rz-ax -f 6.3+2.1)
 .It Fl F
-Read C strings from stdin and output in hexpairs. Useful to load shellcodes
+Read C strings from stdin and output in hex pairs. Useful to load shellcodes
+.It Fl h
+Show usage help message
 .It Fl i
-Dump stdin to C array in stdout (xxd replacement)
+dump as C byte array
```
Comment (XVilka) [inline binrz/man/rz-ax.1]: Same

File: binrz/man/rz-bin.1 [hunk 14]
```diff
@@ -1,95 +1,92 @@
...
 .Bl -tag -width Fl
 .It Fl @ Ar addr
-Show information (symbol, section, import) of the given address
+Show section, symbol, or import at addr
 .It Fl A
-List sub-binaries and their associated arch-bits pairs
+List sub-binaries and their arch-bits pairs
 .It Fl a Ar arch
-Set arch (x86, arm, .. accepts underscore for bits x86_32)
+Set arch (x86, arm, .. or <arch>_<bits>)
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: "architecture"

File: binrz/man/rz-bin.1 [hunk 15]
```diff
@@ -1,95 +1,92 @@
...
 .Ar file
-.Sh DESCRIPTION
-This program allows you to get information about ELF/PE/MZ and CLASS files in a simple way.
-.Pp
-All those commandline flags are also available under the i command in rizin. Type i? for help.
+.Sh OPTIONS
 .Bl -tag -width Fl
 .It Fl @ Ar addr
-Show information (symbol, section, import) of the given address
+Show section, symbol, or import at addr
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: "at the given address"

File: binrz/man/rz-find.1 [hunk 16]
```diff
@@ -1,66 +1,70 @@
...
-Read the keyword to search from the contents of the given file
-.It Fl t Ar to
-Specify the target adddress
-.It Fl X
-Display hexdump of search results
-.It Fl Z
-Display zero-terminated strings results
+Read the contents of the file and use it as a keyword
+.It Fl h
+show this help
```
Comment (XVilka) [inline binrz/man/rz-find.1]: Same

File: binrz/man/rz-run.1 [hunk 17]
```diff
@@ -1,124 +1,85 @@
...
+The program accepts a single argument, which is the filename of the configuration file to run the program.
+.Pp
+It is useful when you have to run a program using long arguments or pass long data to stdin, or things like that usually required for exploiting crackmes :)
 .Sh DIRECTIVES
 .Pp
 The rrz (rz-run) configuration file accepts the following directives, described as key=value entries and comments defined as lines starting with '#'.
 .Bl -tag -width Fl
 .It Ar arg[0-N]
 set value for argument N passed to the program
-.It Ar aslr
```
Comment (XVilka) [inline binrz/man/rz-run.1:25]: Please keep original format for directives, while removing example before

File: binrz/man/rz-test.1 [hunk 18]
```diff
@@ -25,36 +16,42 @@ You need rizin to be available in $PATH.
...
 Show the help/usage message
-.It Fl i
-Interactive mode
-.It Fl n
-Do not run any test, just load/parse them
-.It Fl q
-Quiet mode (use like this: rz-test -qv)
 .It Fl v
-Show version number
+show version
```
Comment (XVilka) [inline binrz/man/rz-test.1]: Same, also original string was better

File: binrz/man/rz-test.1 [hunk 19]
```diff
@@ -25,36 +16,42 @@ You need rizin to be available in $PATH.
...
+.It Fl q
+quiet
 .It Fl V
-Be verbose
+verbose
```
Comment (XVilka) [inline binrz/man/rz-test.1]: Same

**Review by XVilka — CHANGES_REQUESTED** [review]

File: binrz/man/rz-bin.1 [hunk 20]
```diff
@@ -1,95 +1,92 @@
...
 .It Fl F Ar binfmt
 Force to use that bin plugin (ignore header check)
 .It Fl g
-Show all possible information
+Same as -SMZIHVResizcld -SS -SSS -ee (show all info)
 .It Fl G Ar addr
 Load address . offset to header
 .It Fl h
-Show usage help message.
+This help message
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: Not "this", it's a man page, keep the old message here

File: binrz/man/rz-bin.1 [hunk 21]
```diff
@@ -1,95 +1,92 @@
...
+Entrypoint
+.It Fl E
+Globally exportable symbols
+.It Fl f Ar str
+Select sub-bin named str
 .It Fl F Ar binfmt
 Force to use that bin plugin (ignore header check)
 .It Fl g
-Show all possible information
+Same as -SMZIHVResizcld -SS -SSS -ee (show all info)
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: "information"

File: binrz/man/rz-run.1 (hunk 17 above)
Comment (XVilka) [reply binrz/man/rz-run.1:25]: You forgot this one

**Review by XVilka — CHANGES_REQUESTED** [review]
Overall, it looks better now. As something might have slipped my review, let more eyes check first before merging.

File: binrz/man/rz-bin.1 (hunk 21 above)
Comment (wargio) [reply binrz/man/rz-bin.1]: `Same as -SMZIHVResizcld -SS -SSS -ee (show all information)`

File: binrz/man/rz-asm.1 [hunk 22]
```diff
@@ -1,108 +1,78 @@
-.Dd Oct 30, 2018
+.Dd Jan 22, 2024
 .Dt RZ_ASM 1
 .Sh NAME
 .Nm rz-asm
 .Nd rizin assembler and disassembler tool
 .Sh SYNOPSIS
 .Nm rz-asm
-.Op Fl ABdDeEfCLvwrqx
+.Op Fl ACdDehLBvw
```
Comment (ret2libc) [inline binrz/man/rz-asm.1]: what's the reason to remove some of the options?

File: binrz/man/rz-asm.1 [hunk 23]
```diff
@@ -1,108 +1,78 @@
...
-.It Fl .offset
-.It Fl .org
-.It Fl .os
-.It Fl .short
-.It Fl .text
-.It Fl .thumb
-.El
+Use hex dwords instead of hex pairs when assembling.
+.It Fl w
+What's this instruction for? describe opcode
```
Comment (ret2libc) [inline binrz/man/rz-asm.1]: I prefer the original expalanation.
`Describe opcode`.

File: binrz/man/rz-asm.1 [hunk 24]
```diff
@@ -1,108 +1,78 @@
...
-.It Fl .int16
-.It Fl .int32
-.It Fl .int64
-.It Fl .kernel
-.It Fl .offset
-.It Fl .org
-.It Fl .os
-.It Fl .short
-.It Fl .text
-.It Fl .thumb
```
Comment (ret2libc) [inline binrz/man/rz-asm.1]: Why removing these? Aren't we supporting these anymore?

File: binrz/man/rz-ax.1 [hunk 25]
```diff
@@ -1,100 +1,100 @@
...
 .Op [expr] ...
 .Sh DESCRIPTION
 This command is part of the rizin project.
 .Pp
-This command allows you to convert values between positive and negative integer, float, octal, binary and hexadecimal values.
+This command allows you to convert values between positive and negative integers, floats, octal, binary, ternary, and hexadecimal values.
 .Sh OPTIONS
 .Bl -tag -width Fl
+.It Fl l
+Append newline to output (for -E/-D/-r/..)
```
Comment (ret2libc) [inline binrz/man/rz-ax.1]: There is no `-E` nor `-D` option.

Comment (ret2libc) [reply binrz/man/rz-ax.1]: I'm trying `-l` option right now and i'm not sure i understand what it does :/ we should find out and update the doc here.

File: binrz/man/rz-ax.1 [hunk 26]
```diff
@@ -1,100 +1,100 @@
...
-Convert from hex string to character (rz-ax \-s 43 4a 50)
-.It Fl S
-Convert from character to hex string (rz-ax \-S C J P)
+.It Fl k
+Keep the same base as the input data
+.It Fl L
+Convert binary to hexadecimal (rz-ax -L 111111111 outputs 0x1ff)
 .It Fl n
-Show hexpairs from integer value
+Show binary number from integer value
```
Comment (ret2libc) [inline binrz/man/rz-ax.1]: this is wrong
```
rz-ax -n 0x11223344
44332211
```
this is hexpairs, not a binary number.

File: binrz/man/rz-ax.1 [hunk 27]
```diff
@@ -1,100 +1,100 @@
...
-  =8    octal
-  =10   decimal
-  =16   hexadecimal
+=f floating point
+=2 binary
+=3 ternary
+=8 octal
+=10 decimal
+=16 hexadecimal
 .Pp
```
Comment (ret2libc) [inline binrz/man/rz-ax.1]: it was better before.

before:
image

after:
image

**Comment by byteninjaa0** [comment]
i am reverting the `man rz-run` as nothing has been changed all the directives and option is working as expected,
if any change has to be done do let me know

File: binrz/man/rz-asm.1 (hunk 22 above)
Comment (byteninjaa0) [reply binrz/man/rz-asm.1]: it must be wrong in `rz-asm -h` making a commit changing that too , should all the options be present in this?

Comment (ret2libc) [reply binrz/man/rz-asm.1]: I'm not sure tbh... @XVilka ?! do you know if man pages usually have all options listed here?

File: binrz/man/rz-ax.1 (hunk 26 above)
Comment (byteninjaa0) [reply binrz/man/rz-ax.1]: changing it in the `rz-ax -h` also

File: binrz/man/rz-asm.1 (hunk 24 above)
Comment (wargio) [reply binrz/man/rz-asm.1]: i guess they are missing in the help

File: binrz/man/rz-asm.1 (hunk 22 above)
Comment (wargio) [reply binrz/man/rz-asm.1]: those are correct. those are the options without variable, like `-h`

Comment (XVilka) [reply binrz/man/rz-asm.1]: Yes, `man` concatenates those with variables after that string, just run `man rz-asm` and see how it looks.

File: binrz/man/rz-ax.1 [hunk 28]
```diff
@@ -66,13 +86,19 @@ Available variable types are:
   \-int  \->  hex    rz-ax \-77
   \-hex  \->  int    rz-ax 0xffffffb3
   int   \->  bin    rz-ax b30
+  int \-> ternary   rz-ax t42
+  ternary -> int   rz-ax 1010dt
```
Comment (XVilka) [inline binrz/man/rz-ax.1]: I think this should be aligned with others

File: binrz/man/rz-bin.1 [hunk 29]
```diff
@@ -101,73 +98,87 @@ Show less info (no offset/size for -z for ex.)
 .It Fl Q
 Show load address used by dlopen (non-aslr libs)
 .It Fl r
-Show output in rizin format
+Rizin output
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: Return to the old `Show ...` descriptions please

File: binrz/man/rz-bin.1 [hunk 30]
```diff
@@ -101,73 +98,87 @@ Show less info (no offset/size for -z for ex.)
...
 .It Fl R
-Show relocations
+Relocations
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: Same, etc

File: binrz/man/rz-bin.1 [hunk 31]
```diff
@@ -101,73 +98,87 @@ Show less info (no offset/size for -z for ex.)
...
+.It Fl Y Ar fw file
+Calculates all the possibles base address candidates of a firmware bin
 .It Fl z
-Show strings inside .data section (like gnu strings does)
-.It Fl Z
-Guess size of binary program
+Strings (from data section)
 .It Fl zz
-Shows strings from raw bins
+Strings (from raw strings from bin)
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: `Show strings (...`

File: binrz/man/rz-bin.1 [hunk 32]
```diff
@@ -101,73 +98,87 @@ Show less info (no offset/size for -z for ex.)
...
+Extract bins contained in file
+.It Fl X Ar fmt [f] ..
+Package in fat or zip the given files and bins contained in file
+.It Fl Y Ar fw file
+Calculates all the possibles base address candidates of a firmware bin
 .It Fl z
-Show strings inside .data section (like gnu strings does)
-.It Fl Z
-Guess size of binary program
+Strings (from data section)
```
Comment (XVilka) [inline binrz/man/rz-bin.1]: `Show strings (...`

File: binrz/man/rz-find.1 [hunk 33]
```diff
@@ -1,66 +1,70 @@
-.Dd Jun 24, 2020
+.Dd Jan 24, 2024
 .Dt RZ_FIND 1
 .Sh NAME
 .Nm rz-find
 .Nd Advanced commandline hexadecimal editor
```
Comment (XVilka) [inline binrz/man/rz-find.1]: I think the description is wrong, it's not an editor

File: binrz/man/rz-find.1 [hunk 34]
```diff
@@ -1,66 +1,70 @@
...
-.Op Fl t Ar to
-.Op Fl [m|s|e] Ar str
+.Op Fl f/t Ar from/to
+.Op Fl  Ar [e|s|w|S|I] str
 .Op Fl x Ar hex
-.Ar file|dir
+.Op Fl  Ar |file|dir ..
 .Sh DESCRIPTION
-rz-find is a program to find byte patterns into files
+.Nm rz-find
```
Comment (XVilka) [inline binrz/man/rz-find.1:18]: Just like with `rz-ax,` please add a note that its part of the Rizin project. For this and all other commands.

File: binrz/man/rz-gg.1 [hunk 35]
```diff
@@ -1,13 +1,14 @@
-.Dd Sep 30, 2014
+.Dd Jan 24, 2024
 .Dt RZ_GG 1
 .Sh NAME
 .Nm rz-gg
 .Nd rizin frontend for r_egg, compile programs into tiny binaries for x86-32/64 and arm.
```
Comment (XVilka) [inline binrz/man/rz-gg.1]: `RzEgg` is better than `r_egg`, also not only x86 and ARM, it supports more architectures, just say "different architectures".

File: binrz/man/rz-hash.1 [hunk 36]
```diff
@@ -32,53 +28,57 @@ This command can be used to calculate hashes of a certain part of a file or a co
 .Pp
 This is the command used by the 'ph' command of rizin.
 .Bl -tag -width Fl
+.It Fl v
+Shows version number
```
Comment (XVilka) [inline binrz/man/rz-hash.1]: Please be consistent in grammar. `Show`, not `shows` everywhere

File: binrz/man/rz-hash.1 [hunk 37]
```diff
@@ -32,53 +28,57 @@ This command can be used to calculate hashes of a certain part of a file or a co
...
-Select an algorithm for the hashing. Valid values are listed in: rz-hash -L
-.It Fl b Ar blocksize
-Define the block size
-.It Fl c Ar hash
-Compare the computed hash with this one. Allowed only when a single hash is computed.
+Hash algorithm to use; you can specify multiple ones by appending a comma (example: sha1,md4,md5,sha256)
+.It Fl B
+Outputs the calculated value for each block
+.It Fl b Ar size
+Sets the block size
```
Comment (XVilka) [inline binrz/man/rz-hash.1]: Same, `Set` not `sets`, here in other places, `output` not `outputs`, etc, you got the idea.

File: binrz/man/rz-test.1 [hunk 38]
```diff
@@ -1,60 +1,64 @@
-.Dd Mar 31, 2020
+.Dd Jan 24, 2024
 .Dt RZ_TEST 1
 .Sh NAME
 .Nm rz-test
 .Nd rizin regression testsuite
```
Comment (XVilka) [inline binrz/man/rz-test.1:5]: Not just `testsuite` but something like `testsuite running tool`. @wargio @kazarmy do you have better ideas for the description?

File: binrz/man/rz-test.1 [hunk 39]
```diff
@@ -1,60 +1,64 @@
...
-.It Fl n
-Do not run any test, just load/parse them
-.It Fl q
-Quiet mode (use like this: rz-test -qv)
+Show usage help message
 .It Fl v
 Show version number
+.It Fl q
+Quiet.
 .It Fl V
```
Comment (XVilka) [inline binrz/man/rz-test.1:25]: Sounds like an order. `Quiet mode` instead. Also remove the period.

File: binrz/man/rz-test.1 [hunk 40]
```diff
@@ -1,60 +1,64 @@
...
+.It Fl r Ar rizin
+Set path to rizin executable (default is rizin)
+.It Fl m Ar rz-asm
+Set path to rz-asm executable (default is rz-asm)
+.It Fl f Ar file
+File to use for JSON tests (default is bins/elf/crackme0x00b)
+.It Fl C Ar dir
+Chdir before running rz-test (default follows executable symlink + test/new)
+.It Fl t Ar seconds
+Timeout per test (default is 960)
```
Comment (XVilka) [inline binrz/man/rz-test.1]: `default is 960 seconds`

File: librz/main/rz-gg.c [hunk 41]
```diff
@@ -28,7 +28,7 @@ static int usage(int v) {
 			"-h",		""              ,"show this help",
 			"-i",		"[shellcode]"   ,"include shellcode plugin, uses options. see -L",
 			"-I",		"[path]"        ,"add include path",
-			"-k",		"[os]"          ,"operating system's kernel (linux,bsd,osx,w32)",
+			"-k",		"[kernel]"          ,"operating system's kernel (linux,bsd,osx,w32)",
```
Comment (XVilka) [inline librz/main/rz-gg.c]: Please fix the indent

**Review by XVilka — CHANGES_REQUESTED** [review]
Looks much better now! Few things left.

**Comment by XVilka** [comment]
1. Please rebase
2. Could you please also add the man pages directory into the `documentation` key in `.github/labeler.yml`? So that it will mark any changes with the `documentation` label automatically.

File: binrz/man/rz-run.1 [hunk 42]
```diff
@@ -1,12 +1,12 @@
-.Dd Feb 3, 2017
+.Dd Jan 24, 2024
 .Dt RZ_RUN 1
 .Sh NAME
 .Nm rz-run
 .Nd rizin utility to run programs in exotic environments
```
Comment (XVilka) [inline binrz/man/rz-run.1]: Maybe not `exotic` but `custom` instead?

**Comment by wargio** [comment]
all these changes should be applied also to the actual `-h` help message of each utility which required these changes.

File: binrz/man/rz-ax.1 (hunk 28 above)
Comment (byteninjaa0) [reply binrz/man/rz-ax.1]: image
have a look

File: binrz/man/rz-find.1 (hunk 33 above)
Comment (byteninjaa0) [reply binrz/man/rz-find.1:5]: "Find byte patterns in files". how is this?

Comment (byteninjaa0) [reply binrz/man/rz-find.1:5]: image

Comment (byteninjaa0) [reply binrz/man/rz-find.1]: image

File: binrz/man/rz-find.1 (hunk 34 above)
Comment (byteninjaa0) [reply binrz/man/rz-find.1:16]: there is no description in rz-bin should i add one like this 
image

Comment (byteninjaa0) [reply binrz/man/rz-find.1:18]: there is no description in rz-bin should i add one like this 
image

File: binrz/man/rz-find.1 (hunk 33 above)
Comment (XVilka) [reply binrz/man/rz-find.1]: Yes, that would work

File: binrz/man/rz-find.1 (hunk 34 above)
Comment (XVilka) [reply binrz/man/rz-find.1:18]: Yes

**Review by XVilka — APPROVED** [review]
I think it's already a big improvement and can be merged as is. If there is something else left, we can fix it in other PRs.

File: binrz/rz-test/rz-test.c [hunk 43]
```diff
@@ -64,44 +64,28 @@ static int help(bool verbose) {
...
+			" -f [file]    File to use for json tests (default is " JSON_TEST_FILE_DEFAULT ")\n"
+			" -C [dir]     Chdir before running rz-test (default follows executable symlink + test/new\n"
+			" -t [seconds] Timeout per test (default is " TIMEOUT_DEFAULT_STR ")\n"
+			" -o [file]    Output test run information in JSON format to file\n"
+			" -e [dir]     Exclude a particular directory while testing (this option can appear many times)\n"
+			" -s [num]     Number of expected successful tests\n"
+			" -x [num]     Number of expected failed tests"
+			"\n"
+			"Supported test types: @json @unit @fuzz @cmds\n"
+			"OS/Arch for archos tests: " RZ_TEST_ARCH_OS "\n");
```
Comment (wargio) [inline binrz/rz-test/rz-test.c]: this breaks the colors tho.

**Comment by wargio** [comment]
Please fix verbs conjugation in the man pages PR. Some verbs are simple while others use third person. 
They all should be simple - `Show` not `Shows`, etc.. within the C files.

**Comment by XVilka** [comment]
@byteninjaa0, please make those verb conjugation adjustments not only in C files - I can see there are a lot of inconsistencies in the man pages themselves, too.

**Comment by byteninjaa0** [comment]
this too ? 
image

**Comment by wargio** [comment]
yes those should appear in the man pages too

**Comment by byteninjaa0** [comment]
i think that the summary/description of the utilities should be in third person otherwise grammatically it would be wrong, isn't it?

**Review by XVilka — APPROVED** [review]

**Pull request merged**